### ISSUE TITLE
Index out of range

### ISSUE THREAD

**Issue by antonija-nosto** [joeyism/linkedin_scraper#39]
I was testing out this script but I couldn't get it going because of

```
self.about_us = grid.find_elements_by_tag_name("p")[0].text.strip()
IndexError: list index out of range
```

Any comments on how to overcome this issue?

**Comment by joeyism** [comment]
Which profile are you using for this?

**Comment by fhoppe86** [comment]
i got the same error and i think i found the issue, in company.py in row 153 you have: 
grid = driver.find_elements_by_tag_name("section")[3]
but sometimes when you open linkedin you get this cookie policy message: "This website uses cookies to improve service and provide tailored ads. By using this site, you agree to this use. See our Cookie Policy"
if this cookie policy message appears the section has to be 4 instead of 3

i have this in my own script:
try:
		if 'Cookie Policy' in driver.find_elements_by_tag_name("section")[1].text:
			section_id = 4
except:
		section_id = 3

**Comment by joeyism** [comment]
@fhoppe86 oh nice. Can you submit a MR with that and I'll merge it in?

**Comment by joeyism** [comment]
This should be fixed as of [2.5.5](https://github.com/joeyism/linkedin_scraper/releases/tag/2.5.5)
